js/src/product-feed/submission-success-guide/enhanced-conversions/index.js

```diff
@@ -0,0 +1,181 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment, createInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import GuidePageContent from '.~/components/guide-page-content';
+import TrackableLink from '.~/components/trackable-link';
+import useAcceptedCustomerDataTerms from '.~/hooks/useAcceptedCustomerDataTerms';
+import useGoogleAdsEnhancedConversionSettingsURL from '.~/hooks/useGoogleAdsEnhancedConversionSettingsURL';
+import AppSpinner from '.~/components/app-spinner';
+
+const EnhancedConversions = () => {
+	const { acceptedCustomerDataTerms, hasFinishedResolution } =
+		useAcceptedCustomerDataTerms();
+	const url = useGoogleAdsEnhancedConversionSettingsURL();
+
+	let title = __(
+		'Optimize your conversion tracking with Enhanced Conversions',
+		'google-listings-and-ads'
+	);
+
+	if ( acceptedCustomerDataTerms ) {
+		title = __(
+			'Your Enhanced Conversions are almost ready',
+			'google-listings-and-ads'
+		);
+	}
```

Comment:
This could be written as a const:

```suggestion
	const title = acceptedCustomerDataTerms
		? __(
				'Your Enhanced Conversions are almost ready',
				'google-listings-and-ads'
		  )
		: __(
				'Optimize your conversion tracking with Enhanced Conversions',
				'google-listings-and-ads'
		  );
```

Reply:
👍